send.go

```diff
@@ -111,5 +107,4 @@ func IsValidMessageCard(webhookMessage MessageCard) (bool, error) {
 	}
 
 	return true, nil
-
```

Comment:
Should I try to prune all extra whitespace from PRs that I submit? I tend to use it liberally, but I understand that some tooling and projects wish to eliminate it in order to condense the code.